README.md

```diff
@@ -25,6 +25,24 @@ To format the code:
 swift format --in-place --recursive .
 ```
 
+## Testing
+
+Running Localhost tests:
```

Comment:
also, mention that there is a env var, `AWS_CRT_LOCALHOST`, needs to be set to run those tests.